app/main.py

```diff
@@ -34,25 +52,21 @@ async def lifespan(app: FastAPI) -> AsyncIterator[Never]:
 
     """
     await init_db()
-    # Route handlers should access this dictionary to send notifications using
-    # various third-party services, such as AWS, Twilio, etc.
-    app.state.providers = {'aws': ProviderAWS()}
 
     yield  # type: ignore
 
-    app.state.providers.clear()
+    app.enp_state.clear_providers()
```

Comment:
Should this clear all state or just providers for now?

Reply:
Ya - Looked at the docs too. Their examples did show the state being cleared - but I don't understand why.